lib/matplotlib/_constrained_layout.py

```diff
@@ -276,21 +277,35 @@ def _make_layout_margins(fig, renderer, *, w_pad=0, h_pad=0,
 def _make_margin_suptitles(fig, renderer, *, w_pad=0, h_pad=0):
     # Figure out how large the suptitle is and make the
     # top level figure margin larger.
+
+    invTransFig = fig.transFigure.inverted().transform_bbox
```

Comment:
Since you're essentially rewriting the whole function, would you care to make the name PEP-8 compliant?

Reply:
The function name?  Or `invTransFig`?

Reply:
`invTransFig`

Reply:
Done..